src/kOS/Function/Math.cs

```diff
@@ -14,7 +15,7 @@ public override void Execute(SharedObjects shared)
             double argument = GetDouble(PopValueAssert(shared));
             AssertArgBottomAndConsume(shared);
             double result = Math.Abs(argument);
-            ReturnValue = result;
+            ReturnValue = Structure.FromPrimitive(result);
```

Comment:
The `ReturnValue` property definition automatically runs the `value` through `Structure.FromPrimitive`.  Should we be passing the value through it twice, or is it a cheap enough function that it's better to be safe than sorry?  This could effect all of the function's that change this value in the PR.